src/domain/community/virtual-contributor/virtual.contributor.resolver.mutations.ts

```diff
@@ -1,24 +1,173 @@
-import { UseGuards } from '@nestjs/common';
-import { Args, Resolver, Mutation } from '@nestjs/graphql';
+import { Inject, LoggerService, UseGuards } from '@nestjs/common';
+import { Args, Resolver, Mutation, ObjectType, Field } from '@nestjs/graphql';
 import { VirtualContributorService } from './virtual.contributor.service';
 import { CurrentUser, Profiling } from '@src/common/decorators';
 import { GraphqlGuard } from '@core/authorization';
-import { AuthorizationPrivilege } from '@common/enums';
+import { AuthorizationPrivilege, ConfigurationTypes } from '@common/enums';
 import { AgentInfo } from '@core/authentication.agent.info/agent.info';
 import { AuthorizationService } from '@core/authorization/authorization.service';
 import { IVirtualContributor } from './virtual.contributor.interface';
 import {
   DeleteVirtualContributorInput,
   UpdateVirtualContributorInput,
 } from './dto';
+import { ChromaClient } from 'chromadb';
```

Comment:
Please move these two mutations down to be on the AI Server, not on VIrtual Contributor - even if they are going to be shortly removed again.